src/ui/Controllers/HomeController.cs

```diff
@@ -43,7 +43,7 @@ public async Task<ActionResult> Index()
             var userOrganisations = orgs.ToList();
             if (userOrganisations.Count() == 1)
             {
-                return this.RedirectToAction("Show", "Organisation", new { ucasCode = userOrganisations[0].UcasCode });
+                return this.RedirectToAction("Show", "Organisation", new { instCode = userOrganisations[0].UcasCode });
```

Comment:
Can i reject?

Reply:
no

Reply:
that UcasCode is from the api, needs fixing there

Reply:
ok